cmd/stack/destroy.go

```diff
@@ -35,12 +36,16 @@ func Destroy(ctx context.Context) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			s, err := utils.LoadStack(name, stackPath)
 			if err != nil {
-				return err
+				if name == "" {
+					return err
+				}
+				s = &model.Stack{Name: name}
```

Comment:
do you think we should add a log, and maybe ask for re-confirmation? (not sure if there's a risk of deleting the wrong things if the manifest is malformed).